ingestion/functions/README.md

```diff
@@ -119,6 +119,114 @@ execution of code.
 You are free to write the parsers however you like.
 It's best to get inspiration from existing functions though and the only prerequisite for it showing up in the UI once your PR is merged in is that its name must include _"ParsingFunction"_.
 
+### Writing a parser
+
+You can find an example minimal parser in the [parsing/example](/ingestion/functions/parsing/example/README.md) directory, let's look at its structure:
+
+```shell
+$ tree parsing/example
+parsing/example
+├── README.md        # You can document your parser if you want.
+├── __init__.py      # Required to make this a proper python package, usually empty.
+├── example.py       # Write your parsing code here.
+├── example_test.py  # Always add unit tests.
+├── input_event.json # AWS Cloudwatch event used when testing locally, will be described below.
+├── requirements.txt # Any special third-party dependency that your parser requires, this file is required even if it's empty.
+└── sample_data.csv  # Some sample data used in unit tests, usually copied verbatim from a real source.
+```
+
+At minima, a parser must generate a list of cases that conform to the openAPI
+specifications. If you have a local stack running, go to the [OpenAPI UI](http://localhost:3001/api-docs) to check the structure of a `Case` object. Otherwise you can always [check it online](https://curator.ghdsi.org/api-docs/) as well.
```

Comment:
nit: can you set line breaks at 80 characters?

Reply:
as discussed, not easily in vscode it's just sugar display but not actually introducing newlines upon formatting:(